components/automate-cli/cmd/chef-automate/deploy.go

```diff
@@ -50,6 +50,7 @@ var deployCmdFlags = struct {
 	enableChefServer                bool
 	enableDeploymentOrderStressMode bool
 	enableWorkflow                  bool
+	enableDesktop                   bool
```

Comment:
we'd like to avoid having the enable flags. that was the old way

Reply:
what's new the new way, @jaym ?

Reply:
@vjeffrey, Jay and I talked and the new way is a lot easier. With Jay's help, I think we have a solution for all our needs for desktop. 

The only question now is would the below line be acceptable to install automate with desktop? 
`chef-automate deploy --product desktop`

Reply:
i think so! i'll send this to jessica to make sure! 🎉

Reply:
confirmed this flag is 👍 with product